### PR TITLE
docs: organise as TRG folder structure

### PR DESCRIPTION
**Pull request by oyo**
## Description

Organise docs according to TRG folder structure

## Why

Docs weren't structured as TRG 1 suggest.

## Issue

#400

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)

### CODE REVIEW

File: docs/api/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### API Documentation
+
+Go to the [API Hub](https://eclipse-tractusx.github.io/api-hub/)
```
Comment (evegufy) [inline docs/api/README.md:3]:
```suggestion
Please refer to the `docs/api` directory of the [portal-backend repository](https://github.com/eclipse-tractusx/portal-backend) or the [API Hub](https://eclipse-tractusx.github.io/api-hub/portal-backend) for the respective version you are interested in.
```

**Review by evegufy — CHANGES_REQUESTED** [review]
could you please also add a docs/README.md file describing of what the documentation is made up on a high level? [example](https://github.com/eclipse-tractusx/ssi-authority-schema-registry/tree/main/docs)

File: docs/admin/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+### Administration
+
+Sections
+
+- [DB Views and Details](./DB+Views+and+Details/)
```
Comment (evegufy) [inline docs/admin/README.md:5]: could you please fix all places where you use such `+` encodings, they don't work and result in 404 errors
```suggestion
- [DB Views and Details](./DB%20Views%20and%20Details/)
```

**Review by evegufy — CHANGES_REQUESTED** [review]

File: docs/admin/README.md (hunk 2 above)
Comment (oyo) [reply docs/admin/README.md]: done

**Review by evegufy — APPROVED** [review]

**Pull request merged**